eng/testing/tests.mobile.targets

```diff
@@ -10,7 +10,7 @@
     <PublishingTestsRun>true</PublishingTestsRun>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetOS)' == 'Browser'">
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser' And '$(IsTrimmingTest)' != 'true'">
```

Comment:
Is there a better / more descriptive property name that can be used here?

Reply:
do you have a suggestion here? Not sure what could be more descriptive than this since these targets only get imported by our tests, so seems like IsTrimmingTest should be descriptive enough given the context.

Reply:
My original thinking here is the property should be named wrt what it is doing. For example, a bunch of the properties below are simulating what the Blazor WASM SDK defaults for the feature switches. (Not sure what the RunScriptCommand part is doing.)


So the last properties could be behind a property named ‘DefaultBlazorWASMFeatureSwitches’. The idea being if some other project that wasn’t a trimming test needed to shut this off, they could set that property vs adding another condition.